docs/plugin_setup.md

```diff
@@ -1,8 +1,8 @@
 # Setting up your plugin configuration for oAuth
 
-  - Go to the Mattermost Azure Devops plugin configuration page on Mattermost as **System Console > Plugins > Mattermost Azure Devops plugin**.
-  - On the Mattermost Azure Devops plugin configuration page, you need to configure the following:
-    - **Azure Devops API base URL**: Enter the base URL for Azure Devops API (`https://dev.azure.com`).
+  - Go to the Mattermost Azure DevOps plugin configuration page on Mattermost as **System Console > Plugins > Mattermost Azure Devops plugin**.
+  - On the Mattermost Azure DevOps plugin configuration page, you need to configure the following:
+    - **Azure Devops API base URL**: Enter the base URL for Azure DevOps API (`https://dev.azure.com`).
```

Comment:
Ideally, you should do this change here and in the `plugin.json` as well.
~Devops~ DevOps

Reply:
Done